cnxarchive/tests/test_database.py

```diff
@@ -780,8 +780,12 @@ def test_republish_collection(self, cursor):
         collection_ident = cursor.fetchone()[0]
         cursor.connection.commit()
 
-        new_ident = republish_collection(
-            3, collection_ident, testing.fake_plpy)
+        replublished_submittor = "replublished_submittor"
+        replublished_submitlog = "replublished_submitlog"
+
+        new_ident = republish_collection(replublished_submittor,
```

Comment:
@karenc Actually, I read my test code again and I think I am doing it correctly. The previous republish_collectioon method ignores submitter/submitlog. After my change, it takes the submitter/submitlog as arguments and insert them into the correct row. In the test I modified, I checked if the submitter/submitlog is correctly inserted into the row. Correctly inserted data solves the bug mentioned in [this ticket](https://trello.com/c/aXesAJtk/13-improve-full-archive-search-w-in-book-search-technology). So I think my commit is not doing anything wrong besides the typo.